executor/common_linux.h

```diff
@@ -3849,6 +3987,9 @@ static int do_sandbox_none(void)
 #define SYZ_HAVE_SANDBOX_SETUID 1
```

Comment:
It's intentional that `SYZ_NIC_VF` does not work for `do_sandbox_namespace`, right?

Reply:
Correct. It relies on netpci0 interface being created, which is not done in do_sandbox_namespace.